### ISSUE TITLE
aria-label doesn't work when activate dynamic data for button

### ISSUE THREAD

**Issue by leohsiang** [tomusborne/generateblocks#696]
### Description
aria-label doesn't work when activating dynamic data for button block.

### Steps to reproduce
1.  Add a buttons block, and activate the dynamic data option for a button.
2.  Add aria-label in the Advanced section.

#### Expected behavior
aria-label being added to the button's HTML.

#### Actual behavior
aria-label is not added to the button's HTML.

#### References: 
https://generatepress.com/forums/topic/aria-label/#post-2342156

-----

**Comment by JeanPaiva** [comment]
Confirmed the bug in version 1.7

**Comment by tomusborne** [comment]
@JeanPaiva I'm still able to reproduce this. We're missing the `aria-label` attribute in here: https://github.com/tomusborne/generateblocks/blob/master/includes/blocks/class-button.php#L452

This is using the "ARIA LABEL" option in the "Advanced" panel (not custom attributes).
